CHAP/runner.py

```diff
@@ -102,15 +108,22 @@ def run(pipeline_config, interactive=False, logger=None, log_level=None, log_han
             kwargs = {**kwargs, **item[name]}
         else:
             name = item
-        modName, clsName = name.split('.')
-        module = __import__(f'CHAP.{modName}', fromlist=[clsName])
-        obj = getattr(module, clsName)()
-        if log_level:
-            obj.logger.setLevel(log_level)
-        if log_handler:
-            obj.logger.addHandler(log_handler)
-        if logger:
-            logger.info(f'Loaded {obj}')
+        if "users" in name:
+            obj = UserProcessor()
+            clsName = name.split('.')[-1]
+            modName = '.'.join(name.split('.')[:-1])
+            module = __import__(modName, fromlist=[clsName])
+            obj = getattr(module, clsName)()
+        else:
+            modName, clsName = name.split('.')
+            module = __import__(f'CHAP.{modName}', fromlist=[clsName])
+            obj = getattr(module, clsName)()
+            if log_level:
+                obj.logger.setLevel(log_level)
+            if log_handler:
+                obj.logger.addHandler(log_handler)
+            if logger:
+                logger.info(f'Loaded {obj}')
```

Comment:
Why shouldn't UserProcessors' loggers be set up in the same way as other PipelineItems?